tests/components/zwave_js/conftest.py

```diff
@@ -276,6 +276,28 @@ def climate_radio_thermostat_ct100_plus_different_endpoints_state_fixture():
     )
 
 
+@pytest.fixture(name="climate_adc_t3000_state", scope="session")
+def climate_adc_t3000_state_fixture():
+    """Load the climate ADC-T3000 node state fixture data."""
+    return json.loads(load_fixture("zwave_js/climate_adc_t3000_state.json"))
+
+
+@pytest.fixture(name="climate_adc_t3000_missing_setpoint_state", scope="session")
+def climate_adc_t3000_missing_setpoint_state_fixture():
+    """Load the climate ADC-T3000 node state (missing de-humidify setpoint) fixture data."""
+    return json.loads(
+        load_fixture("zwave_js/climate_adc_t3000_missing_setpoint_state.json")
+    )
+
+
+@pytest.fixture(name="climate_adc_t3000_missing_mode_state", scope="session")
+def climate_adc_t3000_missing_mode_state_fixture():
```

Comment:
Instead of adding a fixture json file for each case, maybe it's easy enough to modify the fixture data for a good device and remove the data that should be missing? Each time we load a new fixture file it takes a significant relative amount of time in the test run.

Reply:
Done